src/jsonToGraphQLQuery.test.ts

```diff
@@ -53,14 +59,24 @@ describe('jsonToGraphQLQuery', () => {
               __args: {
                 foo: 'UUID!',
               },
+              something: 'Boolean!',
+              another: {
+                __args: {
+                  foo: 'String!',
+                  bar: 'String!',
+                },
+                zeta: 'Int!',
+              },
             }
           },
         },
       },
     })
 
-    Object.keys(response.variables).forEach(varName => {
-      expect(response.query.split(varName).length - 1).toBeGreaterThanOrEqual(2) //every variable name should be present at least 2 times
+    expect(response.variables).toEqual({
+      foo_0: 'Bar',
+      foo_1: 'Lorem ipsum',
+      bar_0: 'Dolor sit amet',
```

Comment:
in case if there's just 1 occurrence of a variable, maybe let's keep as is? (just `bar`)